tempesta_fw/main.c

```diff
@@ -126,10 +126,8 @@ tfw_cleanup(struct list_head *mod_list)
 
 	tfw_cfg_cleanup(mod_list);
 
-	if (!tfw_runstate_is_reconfig()) {
+	if (!tfw_runstate_is_reconfig())
 		tfw_sg_wait_release();
-		tfw_cli_wait_release();
```

Comment:
As long I remember, this was introduces to be sure that all client related callbacks has finished and all client connections was destroyed. It's required since client connections destruction is asynchronous operations. What guarantees it now?

Reply:
The reasons for this change are explained in https://github.com/tempesta-tech/tempesta/pull/1145#discussion_r249758558.

Reply:
Ok, thanks.